collectors/proc.plugin/multi_metadata.yaml

```
@@ -849,20 +862,23 @@ modules:
       plugin_name: proc.plugin
       module_name: /proc/softirqs
       monitored_instance:
-        name: proc /proc/softirqs
+        name: Software Interrupt Request
```

```suggestion
        name: SoftIRQ statistics
```

Most users who care about this will recognize what the name means.